chrome/content/zotero/xpcom/data/itemFields.js

```diff
@@ -172,7 +173,7 @@ Zotero.ItemFields = new function() {
 		}
 		
 		if (_itemTypeFields[itemTypeID]) {
-			return _itemTypeFields[itemTypeID];
+			return _itemTypeFields[itemTypeID].slice();
```

Comment:
Though at this point, the change is not relevant to the PR, so I'll just remove it from the commits and we can address it later if it's still relevant.